src/clj/rems/api/services/organizations.clj

```diff
@@ -47,14 +51,41 @@
        (find-first (comp #{(:organization/id org)} :organization/id))))
 
 (defn add-organization! [userid org]
-  (organizations/add-organization! userid org))
+  (try
+    (organizations/add-organization! userid org)
+    {:success true
+     :organization/id (:organization/id org)}
+    (catch Exception ex
+      {:success false}
```

Comment:
I think I wanted it originally for the duplicate check but since `add-organization!` now handles it, it has become unnecessary.

Reply:
I think you should be able to do something like this on the postgres side:
```
INSERT INTO organizations ....
ON CONFLICT DO NOTHING
RETURNING id
```

Reply:
Also we don't have to return the `{:success ...` here because it is returned by the `add-organization!` but I think it's the obligation of the service-layer to format the reply so I'll remove it from the db-fn.

Reply:
yeah we're a bit inconsistent in that. sometimes the db layer returns `{:success true}` for convenience so that the service layer can just pass it through

Reply:
This was cleaned a bit as well!